### PR TITLE
ARGO-181 Service status timeline request

### PR DESCRIPTION
**Pull request by kaggis**
Add Model, view controllers for service status timeline request
Use new routing paradigm
Add unit-tests

### CODE REVIEW

**Comment by kaggis** [comment]
please review
/cc @pkoro @makistsantekidis

File: app/statusMetrics/model.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+// DataOutput struct holds the queried data from datastore
+type DataOutput struct {
+	Report            string `bson:"report"`
+	Timestamp         string `bson:"timestamp"`
+	EndpointGroup     string `bson:"endpoint_group"`
+	Service           string `bson:"service"`
+	Hostname          string `bson:"hostname"`
+	Metric            string `bson:"metric"`
+	Status            string `bson:"status"`
+	DateInteger	 	  string `bson:"date_integer"`
```
Comment (pkoro) [inline app/statusMetrics/model.go]: identation

File: app/statusMetrics/model.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package statusMetrics
```
Comment (pkoro) [inline app/statusMetrics/model.go]: add license

File: app/statusServices/statusMetrics_test.go [hunk 3]
```diff
@@ -0,0 +1,339 @@
+/*
```
Comment (pkoro) [inline app/statusServices/statusMetrics_test.go]: maybe rename test file to match with package name.

File: app/statusServices/model.go [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+// xml response related structs
+
+type rootXML struct {
+	XMLName        xml.Name `xml:"root"`
+	EndpointGroups []*endpointGroupXML
+}
+
+type endpointGroupXML struct {
+	XMLName   xml.Name `xml:"Group"`
```
Comment (pkoro) [inline app/statusServices/model.go]: maybe it's better to change `Group` to `group` here and in subsequent structs used by the view component.

**Comment by kaggis** [comment]
- Accept Header handler implemented
- Json output implemented 
- Licenses Added
- Unittest update and clean up

Please review
/cc @pkoro @makistsantekidis

**Comment by pkoro** [comment]
+1

**Comment by dpavlos** [comment]
+1

**Pull request merged**